vnext/Desktop/ABI/TestController.h

```diff
@@ -11,7 +11,9 @@ struct TestController {
 
   static Microsoft::ReactNative::IJSValueReader CreateDynamicReader(Microsoft::ReactNative::IJSValueWriter writer);
   static Microsoft::ReactNative::IJSValueWriter CreateDynamicWriter();
-  static Microsoft::ReactNative::IReactContext CreateTestContext();
+  static Microsoft::ReactNative::IReactContext CreateContext(
+      Microsoft::ReactNative::IReactPropertyBag propertyBag,
```

Comment:
>IReactPropertyBag [](start = 30, length = 17)

Usually we pass the parameters as const& to avoid extra copies and have the generated code to be more efficient.

Reply:
I wasn't aware that that is an option when implementing IDL-defined interfaces, but I see now that cppwinrt generates 'const&' parameters.